cfme/control/explorer/policies.py

```diff
@@ -50,12 +49,7 @@ class EditPolicyEventAssignments(ControlExplorerView):
     cancel_button = Button("Cancel")
     save_button = Button("Save")
 
-    @property
-    def supposed_title(self):
-        if BZ(1531468, forced_streams=["5.9"]).blocks:
-            return 'Editing {} {} "{}" Event Assignments'
-        else:
-            return 'Editing {} {} Policy "{}" Event Assignments'
+    supposed_title = 'Editing {} {} Policy "{}" Event Assignments'
```

Comment:
this was a property?Did you mean to remove it

Reply:
Yes, I meant to remove this property -- the BZ no longer applies here.